tests/integration/records/single_db/test_records_unload_datetime.py

```diff
@@ -0,0 +1,434 @@
+                                               'header-row': False,
+                                               **addl_hints,  # type: ignore
+                                           })
+            csv_text = self.unload(column_name='timestamptz',
+                                   records_format=records_format)
+            self.assertIn(csv_text, [create_sample(datetimeformattz) + "\n",
+                                     create_sample(datetimeformattz).replace('-00', '+00') + "\n",
+                                     create_sample(datetimeformattz).replace('-00', '') +
+                                     ".000000\n",
+                                     # TODO: Should this be necessary?
+                                     create_sample(datetimeformattz) + ".000000\n",
+                                     # TODO: Should this be necessary?
+                                     create_sample(datetimeformattz).replace('-00', '') +
+                                     '.000000+0000\n',
+                                     # TODO: Should this be necessary?
+                                     create_sample(datetimeformattz) +
+                                     f":{SAMPLE_SECOND:02d}.000000\n"
+                                     ],
+                          f"from datetimeformattz {datetimeformattz} and addl_hints {addl_hints}")
+
+    def test_unload_timeonly(self) -> None:
+        self.createTimeTable()
+        for timeonlyformat in TIMEONLY_CASES:
+            pandas_compatible_addl_hints: PartialRecordsHints = {
+                'dateformat': 'YYYY-MM-DD',
+                'datetimeformat': f'YYYY-MM-DD {timeonlyformat}',
+                'datetimeformattz': f'YYYY-MM-DD {timeonlyformat}',
+            }
+            addl_hints: PartialRecordsHints = {}
+            if self.engine.name == 'mysql':
+                # mysql has no exporter defined, so everything is via Pandas
+                addl_hints = pandas_compatible_addl_hints
+                if 'AM' in timeonlyformat:
+                    # TODO: Add a GitHub issue for this
+                    logger.warning('Cannot export this dateformat using Pandas, '
+                                   'which is needed on MySQL--skipping test')
+                    continue
+            elif self.engine.name == 'postgresql':
+                if timeonlyformat in ['HH:MI:SS', 'HH24:MI:SS']:
+                    # ensure we keep other areas of ISO style
+                    addl_hints = {
+                        'dateformat': 'YYYY-MM-DD',
+                        'datetimeformat': 'YYYY-MM-DD HH:MI:SS',
+                        'datetimeformattz': 'YYYY-MM-DD HH:MI:SSOF',
+                    }
+                else:
+                    # We will be using pandas
+                    addl_hints = pandas_compatible_addl_hints
+                    if 'AM' in timeonlyformat:
+                        # TODO: Add a GitHub issue for this
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker